examples/management_policies.ipynb

```diff
@@ -0,0 +1,551 @@
+    "\n",
+    "\n",
+    "[write_policy] = manager.managementPolicy([image_trait_set], PolicyAccess.kWrite, context)\n",
+    "\n",
+    "helpers.display_result(write_policy.traitSet())"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-01-30T18:11:38.164577993Z",
+     "start_time": "2024-01-30T18:11:38.149463152Z"
+    }
+   },
+   "id": "15e1be1277309b62",
+   "execution_count": 33
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Once again we see the `Managed` trait, indicating that entities of this kind can be published. In addition, we see that `PixelBased`, `LocatableContent` and `OCIOColorManaged` are traits with associated properties that can be persisted by the manager.\n",
+    "\n",
+    "But what if we're not capable of providing data for all those traits? There are actually two questions here\n",
+    "\n",
+    "1. For a given trait, which of its properties _must_ be filled in for that trait to be published, i.e. which properties are non-optional?\n",
+    "2. For a given entity, which traits _must_ have their required properties filled in for the entity to be published?\n",
+    "\n",
+    "The first question varies by trait and the only way to know is to look at the documentation for that trait.\n",
+    "\n",
+    "The second question is once again determined by the manager's policy, where we wish to ask the manager for the `kRequired` subset of an entity's traits"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "c6453ae700db6604"
+  },
+  {
+   "cell_type": "code",
+   "outputs": [
+    {
+     "data": {
+      "text/markdown": "> **Result:**\n> `{'openassetio-mediacreation:color.OCIOColorManaged', 'openassetio-mediacreation:content.LocatableContent', 'openassetio-mediacreation:managementPolicy.Managed'}`"
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "[required_policy] = manager.managementPolicy([image_trait_set], PolicyAccess.kRequired, context)\n",
+    "\n",
+    "helpers.display_result(required_policy.traitSet())"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-01-30T18:11:38.164953871Z",
+     "start_time": "2024-01-30T18:11:38.149610878Z"
+    }
+   },
+   "id": "8059d9533bf54deb",
+   "execution_count": 34
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "We can see that the required traits are `LocatableContent` and `OCIOColorManaged` (i.e. the properties of `PixelBased` need not be filled in). Note that _all_ traits of an entity's trait set must be imbued in the published data - the `kRequired` traits are those that must _additionally_ have their required properties set.\n",
+    "\n",
+    "Before performing a render, we should check if the manager has any values that it wants to dictate to the host ahead of time, i.e. we want to know which of the traits can be `resolve`d as `kManagerDriven`, as opposed to driven by the host"
```

Comment:
[Consider][Minor]
The `i.e` seems redundant and even distracting to the point you're making, you're getting into reaction later anyway.
```suggestion
    "Before performing a render, we should check if the manager has any values that it wants to dictate to the host ahead of time."
```

Reply:
I'm not sure what you mean by "getting into reaction later" - do you mean the next section with the publishing workflow?  - I think the link is tenuous.

I'll rewrite to hopefully make it a bit clearer.

Reply:
Reaction as in, the host reacting to the managementPolicy query by letting certain fields be driven by the manager vs driving them themselves.

New version is fine.